backend/reservations/management/commands/create_rooms.py

```diff
@@ -11,7 +11,7 @@
 from reservations.management import getch
 
 logging.basicConfig(stream=sys.stdout, level=roombaht_config.LOGLEVEL)
-logger = logging.getLogger('createStaffAndRooms')
+logger = logging.getLogger(__name__)
```

Comment:
ohh we could probably do this in every module, good call.

Reply:
fixed in other places, left the view_XXX names cause thats more specific